Dockerfile

```diff
@@ -28,14 +30,14 @@ FROM builder AS manager-build
 RUN \
     --mount=type=cache,target=${GOCACHE} \
     --mount=type=cache,target=/go/pkg/mod \
-    GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o out/manager main.go
+    GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build ${LDFLAGS:+-ldflags "$LDFLAGS"} -o out/manager main.go
```

Comment:
i think you need something like 

-ldflags "${LDFLAGS:--X github.com/Azure/eraser/pkg/version.Version=latest}"